src/components/Header/Header.styled.jsx

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+const HeaderWrapper = styled.div`
+  background-color: #1c5476;
```

Comment:
I recommend handling the colors with a theme provider, so changing colors with the light and dark theme becomes easier:

https://styled-components.com/docs/advanced#function-themes